superset/views/base.py

```diff
@@ -206,6 +206,9 @@ def wraps(self: "BaseSupersetView", *args: Any, **kwargs: Any) -> FlaskResponse:
             return json_errors_response(
                 errors=[ex.error], status=ex.status, payload=ex.payload
             )
+        except SupersetErrorsException as ex:
+            logger.warning(ex)
```

Comment:
```suggestion
            logger.warning(ex, exc_info=True)
```
just in case we need to debug this at some point